XiEditor/EditViewController.swift

```diff
@@ -214,7 +236,11 @@ class EditViewController: NSViewController, EditViewDataSource, FindDelegate, Sc
         willScroll(to: scrollView.contentView.bounds.origin)
         editView.gutterCache = nil
         shadowView.updateShadowColor(newColor: theme.shadow)
+        if self.unifiedTitlebar == true {
```

Comment:
see above; any properties of the statusbar related to this setting should be set in along with other changes related to this setting